src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -66,6 +67,8 @@ const AnonymousChannelListScreen = ({route}) => {
         <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
       </View>
 
+      <IncognitoEmptyChat totalChannel={anonChannels.length} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `IncognitoEmptyChat` component is added here, but it seems like it will always be rendered regardless of the number of anonymous channels. If the purpose of this component is to display a message when there are no anonymous channels, you should conditionally render it based on the length of `anonChannels`. Here's how you can do it:

```diff
- <IncognitoEmptyChat totalChannel={anonChannels.length} />
+ {anonChannels.length === 0 && <IncognitoEmptyChat totalChannel={anonChannels.length} />}
```
This way, the `IncognitoEmptyChat` component will only be rendered when there are no anonymous channels.


<!-- This is an auto-generated comment by OSS CodeRabbit -->